app/controllers/unwithdraw_controller.rb

```diff
@@ -1,7 +1,33 @@
 # frozen_string_literal: true
 
 class UnwithdrawController < ApplicationController
-  def index
+  def confirm
     @document = Document.with_current_edition.find_by_param(params[:id])
+
+    if !current_user.has_permission?(User::PRE_RELEASE_FEATURES_PERMISSION)
+      render :index
```

Comment:
This `render` might be brittle, as it relies on us setting all the same instance variable for the `index` action. I know it's not going to be there for that long, but maybe a `raise` would be simpler?

Reply:
I'll change the view from `index.html.erb` to `no_pre_release.html.erb` as this may be clearer, and the only instance variable the view needs is `@document` which is set before this view is rendered and this is only going to be here for a matter of days so think it's okay ?

Reply:
Ah, I see - that's much clearer :-).